src/deserialization.hpp

```diff
@@ -388,7 +388,9 @@ std::tuple<ovms::Status, const typename RequestTraits<ovms::InferenceRequest>::T
     }
     }
     if ((!status.ok() || requestTensorPtr == nullptr)) {
-        SPDLOG_DEBUG("Failed to deserialize request. Validation of request failed");
+        if (extractChoice == ExtractChoice::EXTRACT_INPUT) {
```

Comment:
Why we dont need to log errors when choice is output?

Reply:
As long as we iterate over output map we try to get each output. It fails for requests with no output specified so it generated this error message for most of cases. If output is not specified its extraction will failed. We could leave this message if we would put filtered map of outputs info when doing deserialization.